pipelines/basic/scripts/update-version.sh

```diff
@@ -74,9 +74,14 @@ fi
 
 # Update changelog
 if [[ ! -z $changelog ]]; then
-    info "Updating changelog..."
-    rm -f $changelog && touch $changelog
-    printf "### v$version\n\n##### Breaking Changes\n* None.\n\n##### New Features\n* None.\n\n##### Enhancements\n* None.\n\n##### Fixes\n* None.\n\n##### Notes\n* None." >> $changelog
+    if [[ " ${options[@]} " =~ " --grow-changelog " ]]; then
+        info "Inserting template into changelog..."
+        sed -i "/^# .*/a ## [$new_version] - UNRELEASED\n### Breaking Changes\n- None.\n\n### Added\n- None.\n\n### Changed\n- None.\n\n### Fixed\n- None.\n\n### Removed\n- None.\n\n### Notes\n- None.\n" $changelog
+    else
+        info "Resetting changelog to template..."
+        rm -f $changelog && touch $changelog
+        printf "### v$version\n\n##### Breaking Changes\n* None.\n\n##### New Features\n* None.\n\n##### Enhancements\n* None.\n\n##### Fixes\n* None.\n\n##### Notes\n* None." >> $changelog
```

Comment:
@kgreav Shouldn't this be `new_version`? `incr_version` doesn't update the `version` variable, it sets the `new_version` variable.

Reply:
ah good catch - i'm testing on the python SDK so will update and check there.